files/zh-tw/web/css/css_grid_layout/basic_concepts_of_grid_layout/index.md

```diff
@@ -664,5 +664,3 @@ We can control the order in which items stack up by using the `z-index` property
_Layout/Grid_Template_Areas">Grid template areas</a></li><li><a href="/zh-TW/docs/Web/CSS/CSS_Grid_Layout/Layout_using_Named_Grid_Lines">Layout using named grid lines</a></li><li><a href="/zh-TW/docs/Web/CSS/CSS_Grid_Layout/Auto-placement_in_CSS_Grid_Layout">Auto-placement in grid layout</a></li><li><a href="/zh-TW/docs/Web/CSS/CSS_Grid_Layout/Box_Alignment_in_CSS_Grid_Layout">Box alignment in grid layout</a></li><li><a href="/zh-TW/docs/Web/CSS/CSS_Grid_Layout/CSS_Grid,_Logical_Values_and_Writing_Modes">Grids, logical values and writing modes</a></li><li><a href="/zh-TW/docs/Web/CSS/CSS_Grid_Layout/CSS_Grid_Layout_and_Accessibility">CSS Grid Layout and Accessibility</a></li><li><a href="/zh-TW/docs/Web/CSS/CSS_Grid_Layout/CSS_Grid_and_Progressive_Enhancement">CSS Grid Layout and Progressive Enhancement</a></li><li><a href="/zh-TW/docs/Web/CSS/CSS_Grid_Layout/Realizing_common_layouts_using_CSS_Grid_Layout">Realizing common layouts using grids</a></li></ol></li><li data-default-state="open"><a href="#"><strong>Properties</strong></a><ol><li><a href="/zh-TW/docs/Web/CSS/grid">grid</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-area">grid-area</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-auto-columns">grid-auto-columns</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-auto-flow">grid-auto-flow</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-auto-rows">grid-auto-rows</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-column">grid-column</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-column-end">grid-column-end</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-column-gap">grid-column-gap</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-column-start">grid-column-start</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-gap">grid-gap</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-row">grid-row</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-row-end">grid-row-end</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-row-gap">grid-row-gap</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-row-start">grid-row-start</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-template">grid-template</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-template-areas">grid-template-areas</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-template-columns">grid-template-columns</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-template-rows">grid-template-rows</a></li></ol></li><li data-default-state="open"><a href="#"><strong>Glossary</strong></a><ol><li><a href="/zh-TW/docs/Glossary/Grid">Grid</a></li><li><a href="/zh-TW/docs/Glossary/Grid_lines">Grid lines</a></li><li><a href="/zh-TW/docs/Glossary/Grid_tracks">Grid tracks</a></li><li><a href="/zh-TW/docs/Glossary/Grid_cell">Grid cell</a></li><li><a href="/zh-TW/docs/Glossary/Grid_areas">Grid areas</a></li><li><a href="/zh-TW/docs/Glossary/Gutters">Gutters</a></li><li><a href="/zh-TW/docs/Glossary/Grid_Axis">Grid Axis</a></li><li><a href="/zh-TW/docs/Glossary/Grid_rows">Grid row</a></li><li><a href="/zh-TW/docs/Glossary/Grid_column">Grid column</a></li></ol></li></ol></section>
```

Comment:
This is a sidebar content (we used long long ago), remove this as we have one at the top of this document.